fcs/FSharp.Compiler.Service.ProjectCrackerTool/ProjectCrackerTool.fs

```diff
@@ -464,6 +464,14 @@ module internal ProjectCrackerTool =
 #if !DOTNETCORE
               addMSBuildv14BackupResolution ()
 #endif
+              try
+                // this type is used inside the msbuild xml files for evaluation - if it is not available, it will later fail anyway.
+                // so try to preload it, and fail with an explicit error if that fails.
+                let t = typeof<Microsoft.Build.Utilities.ToolLocationHelper>
```

Comment:
a typeof should trigger an assembly load, and fail with an exception if the assembly can't be found, right?

Since this does _not_ fail, but the CI still fails, the issue seems to be something different ...

Reply:
It didn't say the type is missing. It said something about wrong invocation syntax